frontend/src/components/common/input/input.tsx

```diff
@@ -31,20 +31,28 @@ const Input: FC<Props> = ({
 }) => {
   const { field } = useFormControl({ name, control });
   const hasError = Boolean(errors[name]);
+  const hasLabel = Boolean(label);
   const isTextarea = Boolean(rows);
 
   return (
     <label className={styles.inputLabel}>
-      <span className={styles.span}>{label}</span>
+      <span
+        className={getValidClasses(
+          styles.span,
+          !hasLabel && styles.visuallyHidden,
```

Comment:
```suggestion
          !hasLabel && 'visually-hidden',
```
we have a global class for it